app/com/linkedin/drelephant/ElephantRunner.java

```diff
@@ -48,23 +48,38 @@
   private static final long WAIT_INTERVAL = 60 * 1000;      // Interval between fetches and retries
```

Comment:
That's good. Could you help me to find a better name than `drelephant.work.interval` ?